components/fonts/system_font_service.rs

```diff
@@ -41,26 +42,27 @@ use crate::FontData;
 pub enum FontIdentifier {
     Local(LocalFontIdentifier),
     Web(ServoUrl),
+    Mock(Atom),
```

Comment:
Would it make sense to disable this variant for non-test builds?
```suggestion
    #[cfg(test)] Mock(Atom),
```

Also, instead of having the Mock variant, would it make sense to have a `fn new_mock(id: String) -> LocalFontIdentifier` method for each platform's LocalFontIdentifier impl? It does involve adding more code.

Reply:
I did consider that but the issue that the `LocalFontIdentifier` for Windows is currently this https://docs.rs/dwrote/latest/dwrote/struct.FontDescriptor.html which doesn't really support arbitrary local files at the moment.

Reply:
> Would it make sense to disable this variant for non-test builds?

I went back and forth on this, but enabled it for non-test builds so that it was easier to catch issues earlier in development, rather than only when running the test build. In addition it made matches a bit easier, since we could avoid wildcards (though looking over things, I didn't go back and do this yet).

Reply:
I feel like a decent followup to this would be to see if we can figure out how to make the `LocalFontIdentifier` for Windows a path and index.